src/main/java/com/microsoft/spring/data/gremlin/conversion/script/GremlinScriptLiteral.java

```diff
@@ -35,4 +35,9 @@
      * Generate the update query from source (Vertex, Edge or Graph).
      */
     List<String> generateUpdateScript(GremlinSource source);
+
+    /**
+     * Generate the update query from source (Vertex, Edge or Graph).
+     */
+    List<String> generateFindAllScript(GremlinSource source);
```

Comment:
Comment is inconsistent with the method name.

Reply:
Updated.